src/store/modules/dashboard/type.ts

```diff
@@ -0,0 +1,36 @@
+
+export const VIEWERS_TYPE = Object.freeze({
+    PUBLIC: 'PUBLIC',
+    PRIVATE: 'PRIVATE',
+} as const);
+
+export type ViewersType = typeof VIEWERS_TYPE[keyof typeof VIEWERS_TYPE];
+
+export interface DashboardState {
+    domainItems: DomainDashboardItem[];
+    projectItems: ProjectDashboardItem[];
```

Comment:
suggestion: make it undefinable or nullable.

It is often necessary to differentiate between initial loading and other loading. 
It seems safer to distinguish between cases where there is no stale data (first loading) and cases where there is stale data, so that all cases can be dealt with.

```suggestion
    domainItems?: DomainDashboardItem[];
    projectItems?: ProjectDashboardItem[];
```

or

```suggestion
    domainItems: DomainDashboardItem[]|null;
    projectItems: ProjectDashboardItem[]|null;
```

Reply:
Thanks!!! I'm learning..!